.pipelines/nightly.yaml

```diff
@@ -21,3 +21,8 @@ jobs:
       osTypes:
       - "linux"
       - "windows"
+  - template: soak-tests-template.yaml
+    parameters:
+      clusterConfigs:
+      - "csi-secrets-store-soak-linux-aks"
+      - "csi-secrets-store-soak-win-aks"
```

Comment:
where are these configs?

Reply:
They are downloaded as part of soak tests: https://github.com/Azure/secrets-store-csi-driver-provider-azure/pull/396/files#diff-22792ae5cdc74eefcbce99909d002257cadcf0328af36b1995eb1c5ac7dbc657R33-R45